core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -399,7 +399,7 @@ public Handle open() {
      * @throws X any exception thrown by the callback
      */
     public <R, X extends Exception> R inTransaction(final TransactionIsolationLevel level, final HandleCallback<R, X> callback) throws X {
-        return withHandle(handle -> handle.<R, X>inTransaction(level, callback));
+        return withHandle(handle -> handle.inTransaction(level, callback));
```

Comment:
I'm not certain, but I think `<R, X>` parameters might be required to compile in more recent JDKs.